### ISSUE TITLE
Bug: AttributeError: module 'enum' has no attribute 'IntFlag' #1995

### ISSUE THREAD

**Issue by Casyfill** [manrajgrover/halo#127]
## Description
On installing halo a few packages start  complaining on `enum` (see error text below)

### System settings

- Operating System: MacOS 10.14.3 (18D109)
- Terminal in use: iterm2, bash
- Python version: 3.6, 3.7
- Halo version: 0.0.23
- `pip freeze` output:
```
apipkg==1.5
appdirs==1.4.3
appnope==0.1.0
asciimatics==1.11.0
asn1crypto==0.24.0
atomicwrites==1.3.0
attrs==19.1.0
azure-common==1.1.20
azure-storage-blob==2.0.1
azure-storage-common==2.0.0
backcall==0.1.0
black==19.3b0
bleach==3.1.0
boto3==1.9.134
botocore==1.12.134
certifi==2019.3.9
cffi==1.12.3
chardet==3.0.4
Click==7.0
click-plugins==1.1.1
cligj==0.5.0
colorama==0.3.9
configobj==5.0.6
configparser==3.7.4
contextlib2==0.5.5
coverage==4.5.3
cryptography==2.6.1
cursor==1.2.0
cycler==0.10.0
decorator==4.4.0
defusedxml==0.6.0
descartes==1.1.0
distro==1.4.0
docutils==0.14
dulwich==0.19.11
dvc==0.40.2
entrypoints==0.3
enum34==1.1.6
envkey==1.2.4
execnet==1.6.0
Fiona==1.8.6
future==0.17.1
GDAL==2.4.1
geopandas==0.5.0
gitdb2==2.0.5
GitPython==2.1.11
grandalf==0.6
halo==0.0.23
humanize==0.5.1
hypothesis==4.17.2
idna==2.8
ijson==2.3
inflect==2.1.0
ipykernel==5.1.0
ipython==7.5.0
ipython-genutils==0.2.0
ipywidgets==7.4.2
jedi==0.13.3
Jinja2==2.10.1
jmespath==0.9.4
joblib==0.13.2
jsonpath-ng==1.4.3
jsonschema==2.6.0
jupyter==1.0.0
jupyter-client==5.2.4
jupyter-console==6.0.0
jupyter-core==4.4.0
jupyterlab==0.35.6
jupyterlab-server==0.2.0
kiwisolver==1.1.0
lockfile==0.12.2
log-symbols==0.0.12
luigi==2.8.3
mapclassify==2.0.1
MarkupSafe==1.1.1
matplotlib==3.0.3
mistune==0.8.4
more-itertools==7.0.0
munch==2.3.2
nanotime==0.5.2
nbconvert==5.5.0
nbformat==4.4.0
networkx==2.3
notebook==5.7.8
numpy==1.16.3
pandas==0.24.2
pandocfilters==1.4.2
parso==0.4.0
pep8==1.7.1
pexpect==4.7.0
pickleshare==0.7.5
Pillow==6.0.0
pluggy==0.11.0
ply==3.11
prometheus-client==0.6.0
prompt-toolkit==2.0.9
ptyprocess==0.6.0
py==1.8.0
py-cpuinfo==5.0.0
pyasn1==0.4.5
pycparser==2.19
pycryptodomex==3.8.1
pyfiglet==0.8.post1
Pygments==2.4.0
PyJWT==1.7.1
pymssql==2.1.4
PyMySQL==0.9.3
pyOpenSSL==19.0.0
pyparsing==2.4.0
pyproj==1.9.6
PySocks==1.6.8
pytest==4.4.0
pytest-benchmark==3.2.2
pytest-cache==1.0
pytest-cov==2.6.1
pytest-pep8==1.0.6
python-daemon==2.2.3
python-dateutil==2.8.0
python-dotenv==0.10.2
python-http-client==3.1.0
pytz==2019.1
PyYAML==5.1
pyzmq==18.0.0
qtconsole==4.4.4
requests==2.21.0
Rtree==0.8.3
ruamel.yaml==0.15.94
s3fs==0.2.1
s3transfer==0.2.0
schema==0.7.0
scikit-learn==0.21.1
scipy==1.2.1
Send2Trash==1.5.0
sendgrid==5.6.0
Shapely==1.6.4.post1
sip==4.19.13
six==1.11.0
smmap2==2.0.5
snowflake-connector-python==1.8.0
snowflake-sqlalchemy==1.1.12
spinners==0.0.23
SQLAlchemy==1.3.3
termcolor==1.1.0
terminado==0.8.2
testpath==0.4.2
toml==0.10.0
tornado==5.1.1
tqdm==4.31.1
traitlets==4.3.2
treelib==1.5.5
urllib3==1.24.2
wcwidth==0.1.7
webencodings==0.5.1
Werkzeug==0.15.2
widgetsnbextension==3.4.2
XlsxWriter==1.1.8
zc.lockfile==1.4
```

### Error
```
Failed to import the site module
Traceback (most recent call last):
  File "/Users/philippk/anaconda3/envs/py36/lib/python3.6/site.py", line 541, in <module>
    main()
  File "/Users/philippk/anaconda3/envs/py36/lib/python3.6/site.py", line 522, in main
    known_paths = addusersitepackages(known_paths)
  File "/Users/philippk/anaconda3/envs/py36/lib/python3.6/site.py", line 282, in addusersitepackages
    user_site = getusersitepackages()
  File "/Users/philippk/anaconda3/envs/py36/lib/python3.6/site.py", line 258, in getusersitepackages
    user_base = getuserbase() # this will also set USER_BASE
  File "/Users/philippk/anaconda3/envs/py36/lib/python3.6/site.py", line 248, in getuserbase
    USER_BASE = get_config_var('userbase')
  File "/Users/philippk/anaconda3/envs/py36/lib/python3.6/sysconfig.py", line 608, in get_config_var
    return get_config_vars().get(name)
  File "/Users/philippk/anaconda3/envs/py36/lib/python3.6/sysconfig.py", line 587, in get_config_vars
    import _osx_support
  File "/Users/philippk/anaconda3/envs/py36/lib/python3.6/_osx_support.py", line 4, in <module>
    import re
  File "/Users/philippk/anaconda3/envs/py36/lib/python3.6/re.py", line 142, in <module>
    class RegexFlag(enum.IntFlag):
AttributeError: module 'enum' has no attribute 'IntFlag'
```
 
## Steps to recreate
```
pip install dvc
pip install halo

dvc init (or anything else)
```
see https://github.com/iterative/dvc/issues/1995

**Comment by manrajgrover** [comment]
@Casyfill Thanks for opening this issue!

This must be because of `enum34` being installed in Python3.6 which includes enum as a standard package. This dependency should be conditionally included which was fixed in one of the dependencies of halo but not released.

I've made a release today [`v0.0.25`](https://pypi.org/project/halo/) which should fix this issue. Kindly check and let me know if you're still facing it.
